RecoHI/Configuration/python/customise_PPwithHI.py

```diff
@@ -136,12 +136,47 @@ def storeCaloTowersAOD(process):
         process.AODSIMoutput.outputCommands.extend(['keep *_towerMaker_*_*'])
 
     return process
+	
+# Add rhoProducer to AOD event content
+def addRhoProducer(process):
+    process.load('RecoJets.JetProducers.kt4PFJets_cfi')
+    process.load('RecoHI.HiJetAlgos.hiFJRhoProducer')
+    process.load('RecoHI.HiJetAlgos.hiFJGridEmptyAreaCalculator_cff')
+
+    process.kt4PFJets.doAreaFastjet = True
+    process.kt4PFJets.jetPtMin      = cms.double(0.0)
```

Comment:
@dgulhan  - could these changes go directly into the chi for kt4PFJets?
If not, the module should be cloned instead of modified in place